### PR TITLE
Initialisation

### PR DESCRIPTION
**Pull request by camillaume**

### CODE REVIEW

**Review by LuigiMenez — APPROVED** [review]
No reviewed to SM

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
not reviewed by SM

**Review by n-faycel — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
It's good

File: frontend/src/GlobalStyle.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyle = createGlobalStyle`
+
+
```
Comment (LoicBrassart) [inline frontend/src/GlobalStyle.jsx:5]: From my experience, this means that Prettier and ESLint are *not* configured as needed

File: package.json [hunk 2]
```diff
@@ -17,5 +17,9 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "styled-components": "^5.3.5",
+    "styled-reset": "^4.3.4"
```
Comment (LoicBrassart) [inline package.json]: Nope: you don't need these packages at the project root ;-)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Please fix these minor Evils

**Comment by LoicBrassart** [comment]
@LuigiMenez Beware, there were things to spot in this PR, especially the packages in the root package.json ;-)

**Review by n-faycel — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
No more Evils !

**Pull request merged**